src/_pytest/compat.py

```diff
@@ -1,12 +1,18 @@
 """
 python version compatibility code
 """
+import sys  # isort:skip
+
+if sys.version_info < (3, 5, 2):
+    TYPE_CHECKING = False  # type: bool
+else:
+    from typing import TYPE_CHECKING
```

Comment:
move this below `from _pytest.outcomes ...`? then you don't need to skip isort and E402

Reply:
It was necessery when still imported from `outcomes` (which I've worked around instead, https://github.com/pytest-dev/pytest/pull/6435/files#diff-ced80877d8a5ec1967da20023d71160fR11).  Cleaned up.